pkg/crc/machine/machine.go

```diff
@@ -240,7 +240,7 @@ func Start(startConfig StartConfig) (StartResult, error) {
 	logging.Debug("Waiting until ssh is available")
 	if err := cluster.WaitForSsh(sshRunner); err != nil {
 		result.Error = err.Error()
-		return *result, errors.New("Failed to connect to the CRC VM with SSH")
+		return *result, errors.New("Failed to connect to the CRC VM with SSH -- host might be unreachable")
```

Comment:
Fwiw, "host" can be confusing here, the host is the machine running the VM, while it's the VM which is unreachable.